2025-09-21 18:39:51|INFO    |root                            |BAZARR Running cache maintenance|
2025-09-21 18:43:10|INFO    |root                            |Scheduler will use this timezone: America/Denver|
2025-09-21 18:43:12|INFO    |waitress                        |BAZARR is started and waiting for requests on: http://***.***.***.***:6767|
2025-09-21 18:43:12|INFO    |waitress                        |BAZARR is started and waiting for requests on: http://[::]:6767|
2025-09-21 19:01:31|ERROR   |waitress                        |Exception while serving /bazarr/api/socket.io/|'Traceback (most recent call last):\n  File "/app/bazarr/bin/bazarr/../libs/waitress/channel.py", line 428, in service\n    task.service()\n  File "/app/bazarr/bin/bazarr/../libs/waitress/task.py", line 168, in service\n    self.execute()\n  File "/app/bazarr/bin/bazarr/../libs/waitress/task.py", line 436, in execute\n    app_iter = self.channel.server.application(environ, start_response)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/app/bazarr/bin/bazarr/../libs/waitress/proxy_headers.py", line 64, in translate_proxy_headers\n    return app(environ, start_response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1498, in __call__\n    return self.wsgi_app(environ, start_response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/app/bazarr/bin/bazarr/../libs/flask_socketio/__init__.py", line 42, in __call__\n    return super().__call__(environ, start_response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/app/bazarr/bin/bazarr/../libs/engineio/middleware.py", line 63, in __call__\n    return self.engineio_app.handle_request(environ, start_response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/app/bazarr/bin/bazarr/../libs/socketio/server.py", line 434, in handle_request\n    return self.eio.handle_request(environ, start_response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/app/bazarr/bin/bazarr/../libs/engineio/server.py", line 305, in handle_request\n    socket = self._get_socket(sid)\n             ^^^^^^^^^^^^^^^^^^^^^\n  File "/app/bazarr/bin/bazarr/../libs/engineio/base_server.py", line 242, in _get_socket\n    raise KeyError(\'Session is disconnected\')\nKeyError: \'Session is disconnected\''|
